.github/workflows/dev-test.yml

```diff
@@ -0,0 +1,140 @@
+# Run tests against developer push
+
+# Runs jobs on linux and windows to maintain compatibility.
+# The main repo is checked out into a sub-directory because an optional
+# second repo may be checked out for testing.
+
+name: dev-test
+
+on:
+  push:
+    branches-ignore: [ master, main ]
```

Comment:
Why ignore `main`?

Reply:
My workflow, which may not be correct, is to rebase my main branch from upstream after PRs have been merged.  Since we (I) don't develop on main, it seemed like a waste to run the workflow when the main was pushed to github.  I am open to change or suggestions to take another look at some doc if this is not the way to do things. 🙄

Reply:
Okay. I guess we already have the merge commit tested when merging PRs, so we do not _strictly_ need a CI build.